corehq/apps/users/views/mobile/users.py

```diff
@@ -1128,10 +1128,21 @@ def user_download_job_poll(request, domain, download_id, template="hqwebapp/part
     return render(request, template, context)
 
 
-class DownloadUsersStatusView(BaseManageCommCareUserView):
+class DownloadUsersStatusView(BaseUserSettingsView):
     urlname = 'download_users_status'
     page_title = ugettext_noop('Download Users Status')
 
+    @method_decorator(require_can_edit_or_view_commcare_users)
```

Comment:
I've seen `@method_decorator(require_can_edit_or_view_commcare_users, name='dispatch')` as a decorator on the class to do this in cases like this where there are no other changes to the context of the method itself. I think I like that better personally, but I don't know that we have a standard around that or anything